build.gradle

```diff
@@ -132,17 +132,17 @@ dependencies {
         exclude group: 'xerces', module: 'xmlParserAPIs'
         exclude group: 'xerces', module: 'xercesImpl'
     }
-    api('org.apache.httpcomponents:httpclient:4.5.13') {
+    api('org.apache.httpcomponents:httpclient:4.5.14') {
         exclude group: 'commons-logging', module: 'commons-logging'
     }
-    api 'org.apache.httpcomponents:httpcore:4.4.14'
+    api 'org.apache.httpcomponents:httpcore:4.4.16'
     api 'commons-codec:commons-codec:1.16.0'
 
     // GCP
     api 'com.jcraft:jsch:0.1.55'
-    api 'com.google.apis:google-api-services-compute:v1-rev20221026-2.0.0'
-    api 'com.google.auth:google-auth-library-oauth2-http:1.13.0'
-    api 'com.google.cloud:google-cloud-os-login:2.27.0'
+    api 'com.google.apis:google-api-services-compute:v1-rev235-1.25.0'
```

Comment:
v1-rev235-1.25.0 is very old version than already using. Latest version is v1-rev20231031-2.0.0